### DISCUSSION TITLE
Why database selection needed during Dashboard Import?

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#12879] [Q&A / Help]
Does anyone know the use of Database dropdown in ‘Import Dashboard’ view of Superset 1.0.0? It was not there in 0.36, and I can’t understand the use of it.

![image](https://user-images.githubusercontent.com/74634977/106538005-cf95a280-653e-11eb-851f-9b20be958c6a.png)

I got this response from Euginea in Slack:

> It is to select the database that has the datasets that the dashboard chart use. To import a dashboard, you only need to have the database connection; the import creates the datasets and the charts.

But I still have doubts:
1. Which database should the user select if the dashboard contains charts from 2 or more databases?
2. Even if user selects a database which is not used in the dashboard, the import works fine.

So I am not sure if it this dropdown selection is really used or not. If there is no special use case, then maybe we can remove it again.

**Comment by yousoph** [comment]
The newest version of imports (set feature flag VERSIONED_EXPORTS = True to enable it) no longer requires a database selection when importing 😊

**Comment by kamalkeshavani-aiinside** [comment]
@yousoph Thank you for the comment. I tried and confirmed that it is better. This new import method is only supporting new zip import, so for backward compatibility, the default version of import will continue for some time I guess.
